### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by mwellick**

### CODE REVIEW

File: services/cinema_hall.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> CinemaHall:
```
Comment (EdAlekseiev) [inline services/cinema_hall.py:4]: It returns `QuerySet` of `CinemaHall`. Fix type hint for return values

File: services/movie.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> Movie:
```
Comment (EdAlekseiev) [inline services/movie.py:5]: It returns `QuerySet` of `Movie`. Fix type hint for return values

File: services/movie.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
```
Comment (EdAlekseiev) [inline services/movie.py:4]: If an argument can be None, you must show it in the type hint

File: services/movie.py (hunk 2 above)
Comment (EdAlekseiev) [inline services/movie.py:5]:
```suggestion
def get_movies(genres_ids: list[int] | None= None,
               actors_ids: list[int] | None = None) -> Movie:

```

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

File: services/movie.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+def get_movie_by_id(movie_id: int) -> models.QuerySet:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(movie_title: str,
+                 movie_description: str,
+                 genres_ids: Optional[list[int]] = None,
+                 actors_ids: Optional[list[int]] = None,
+                 ) -> models.QuerySet:
```
Comment (Bohdan-Salamakha) [inline services/movie.py:27]:
```suggestion
def create_movie(
                 movie_title: str,
                 movie_description: str,
                 genres_ids: Optional[list[int]] = None,
                 actors_ids: Optional[list[int]] = None,
                 ) -> models.QuerySet:

```

File: services/movie_session.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+def get_movie_session_by_id(movie_session_id: int) -> models.QuerySet:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(session_id: Optional[int] = None,
+                         show_time: Optional[datetime] = None,
+                         movie_id: Optional[int] = None,
+                         cinema_hall_id: Optional[int] = None
+                         ) -> models.QuerySet:
```
Comment (Bohdan-Salamakha) [inline services/movie_session.py:38]:
```suggestion
def update_movie_session(
                         session_id: Optional[int] = None,
                         show_time: Optional[datetime] = None,
                         movie_id: Optional[int] = None,
                         cinema_hall_id: Optional[int] = None
                         ) -> models.QuerySet:

```

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
fix flake8 errors

**Review by Bohdan-Salamakha — APPROVED** [review]

**Pull request closed without merging**